docs/docs/additional_resources/arxiv_references.mdx

```diff
@@ -388,7 +391,7 @@ https://github.com/AGI-Edgerunners/Plan-and-Solve-Prompting.
 - **URL:** http://arxiv.org/abs/2304.08485v2
 - **LangChain:**
 
-   - **Cookbook:** [Semi_structured_and_multi_modal_RAG](https://github.com/langchain-ai/langchain/blob/master/cookbook/Semi_structured_and_multi_modal_RAG.ipynb), [Semi_structured_multi_modal_RAG_LLaMA2](https://github.com/langchain-ai/langchain/blob/master/cookbook/Semi_structured_multi_modal_RAG_LLaMA2.ipynb)
```

Comment:
out of curiosity why is this one removed?

Reply:
@ccurme Let me see the logs... Got it. The script checks if links are opening OK, but this link had a problem, it was too slow. I'm regenerating the page to see if this persists.